submodules/power_sap_instance_init_sles/variables.tf

```diff
@@ -0,0 +1,52 @@
+variable "bastion_public_ip" {
+  description = "Public IP of Bastion Host"
```

Comment:
It's a "jump box" rather than an actual bastion host. Bastion host implies specific tech like teleport for instance. The powervs infra module uses the name "access_host_or_ip" - could we align?

Reply:
next PRs will introduce the usage. will fix then.